libraries/stumbler/src/main/java/org/mozilla/mozstumbler/service/stumblerthread/scanners/ScanManager.java

```diff
@@ -176,10 +171,8 @@ public synchronized void startScanning(Context ctx) {
         mLocationChangeSensor.start();
 
         if (mMotionSensor == null) {
-            if (mDetectMotionReceiver != null) {
-                LocalBroadcastManager.getInstance(mAppContext).registerReceiver(mDetectMotionReceiver,
-                        new IntentFilter(MotionSensor.ACTION_USER_MOTION_DETECTED));
-            }
+            LocalBroadcastManager.getInstance(mAppContext).registerReceiver(mDetectMotionReceiver,
```

Comment:
Maybe we changed the way it was instantiated, but it can't be null now, it is declared like:
final Thing var = new Thing()